### PR TITLE
Copy in monitoring tools from toolbox

### PR DESCRIPTION
**Pull request by AdamHawtin**
This repo is being split out of [census-rm-toolbox](https://github.com/ONSdigital/census-rm-toolbox/), this PR aims to copy in the monitoring tools from the toolbox as they are.

https://trello.com/c/XHfzQUze/1257-split-cloudshell-utilities-and-monitoring-tools-out-of-census-rm-toolbox

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    DB_USESSL = os.getenv('DB_USESSL', '')
+    DB_ACTION_CERTIFICATES = os.getenv('DB_ACTION_CERTIFICATES',
+                                       (" sslmode=verify-ca sslrootcert=/home/toolbox/.postgresql-action/root.crt "
+                                        "sslcert=/home/toolbox/.postgresql-action/postgresql.crt "
+                                        "sslkey=/home/toolbox/.postgresql-action/postgresql.key"))
+    CASEAPI_HOST = os.getenv('CASEAPI_HOST', 'localhost')
+    CASEAPI_PORT = os.getenv('CASEAPI_PORT', '8161')
+    EXCEPTIONMANAGER_HOST = os.getenv('EXCEPTIONMANAGER_HOST', 'localhost')
+    EXCEPTIONMANAGER_PORT = os.getenv('EXCEPTIONMANAGER_PORT', '8666')
+    EXCEPTIONMANAGER_URL = f'http://{EXCEPTIONMANAGER_HOST}:{EXCEPTIONMANAGER_PORT}'
```
Comment (NickGrantX) [inline config.py]: don't need exception manager?

File: config.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    QID_MODULUS = os.getenv('QID_MODULUS', '1')
+    QID_FACTOR = os.getenv('QID_FACTOR', '1')
```
Comment (NickGrantX) [inline config.py]: don't need these

File: config.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    BULK_WORKING_DIRECTORY = Path(os.getenv('BULK_WORKING_DIRECTORY', '/tmp'))
```
Comment (NickGrantX) [inline config.py]: don't need this

File: config.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+    ENVIRONMENT = os.getenv('ENVIRONMENT')
+
+    EVENTS_EXCHANGE = os.getenv('EVENTS_EXCHANGE', 'events')
+
+
+class TestConfig(Config):
+    COLLECTION_EXERCISE_ID = os.getenv('COLLECTION_EXERCISE_ID', '88960972-38a5-438f-b543-cd9bb464f49a')
+    ACTION_PLAN_ID = os.getenv('ACTION_PLAN_ID', '88960972-38a5-438f-b543-cd9bb464f49a')
```
Comment (NickGrantX) [inline config.py]: Don't need these

File: config.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+
+if Config.ENVIRONMENT == 'TEST':
+    Config = TestConfig
```
Comment (NickGrantX) [inline config.py]: Don't need this

File: config.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    RABBITMQ_PORT = os.getenv('RABBITMQ_SERVICE_PORT', '6672')
+    RABBITMQ_HTTP_PORT = os.getenv('RABBITMQ_HTTP_PORT', '16672')
+    RABBITMQ_VHOST = os.getenv('RABBITMQ_VHOST', '/')
+    RABBITMQ_EXCHANGE = os.getenv('RABBITMQ_EXCHANGE', '')
+    RABBITMQ_USER = os.getenv('RABBITMQ_USER', 'guest')
+    RABBITMQ_PASSWORD = os.getenv('RABBITMQ_PASSWORD', 'guest')
+    DB_USERNAME = os.getenv('DB_USERNAME', 'postgres')
+    DB_PASSWORD = os.getenv('DB_PASSWORD', 'postgres')
+    DB_HOST = os.getenv('DB_HOST', 'localhost')
+    DB_HOST_WRITE = os.getenv('DB_HOST_RW', 'localhost')
```
Comment (NickGrantX) [inline config.py]: Don't need this

File: config.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    DB_HOST_ACTION = os.getenv('DB_HOST_ACTION', 'localhost')
+    DB_PORT = os.getenv('DB_PORT', '6432')
+    DB_PORT_WRITE = os.getenv('DB_PORT_RW', '6432')
```
Comment (NickGrantX) [inline config.py]: Don't need this

File: config.py [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    DB_NAME = os.getenv('DB_NAME', 'postgres')
+    DB_NAME_WRITE = os.getenv('DB_NAME_RW', 'postgres')
```
Comment (NickGrantX) [inline config.py]: Don't need this

File: config.py [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    DB_USESSL = os.getenv('DB_USESSL', '')
+    DB_ACTION_CERTIFICATES = os.getenv('DB_ACTION_CERTIFICATES',
+                                       (" sslmode=verify-ca sslrootcert=/home/toolbox/.postgresql-action/root.crt "
+                                        "sslcert=/home/toolbox/.postgresql-action/postgresql.crt "
+                                        "sslkey=/home/toolbox/.postgresql-action/postgresql.key"))
+    CASEAPI_HOST = os.getenv('CASEAPI_HOST', 'localhost')
+    CASEAPI_PORT = os.getenv('CASEAPI_PORT', '8161')
```
Comment (NickGrantX) [inline config.py]: Don't need these

File: config.py [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+    EXCEPTIONMANAGER_HOST = os.getenv('EXCEPTIONMANAGER_HOST', 'localhost')
+    EXCEPTIONMANAGER_PORT = os.getenv('EXCEPTIONMANAGER_PORT', '8666')
+    EXCEPTIONMANAGER_URL = f'http://{EXCEPTIONMANAGER_HOST}:{EXCEPTIONMANAGER_PORT}'
+    QID_MODULUS = os.getenv('QID_MODULUS', '1')
+    QID_FACTOR = os.getenv('QID_FACTOR', '1')
+
+    BULK_WORKING_DIRECTORY = Path(os.getenv('BULK_WORKING_DIRECTORY', '/tmp'))
+
+    ENVIRONMENT = os.getenv('ENVIRONMENT')
```
Comment (NickGrantX) [inline config.py]: Don't need this

File: utilities/rabbit_helper.py [hunk 11]
```diff
@@ -0,0 +1,14 @@
+from utilities.rabbit_context import RabbitContext
+
+
+def add_test_queue(binding_key, exchange_name, queue_name, exchange_type='topic'):
```
Comment (NickGrantX) [inline utilities/rabbit_helper.py]: Don't need this

**Review by NickGrantX — CHANGES_REQUESTED** [review]
Lots of stuff that can be removed

**Review by NickGrantX — APPROVED** [review]

**Comment by AdamHawtin** [comment]
[Split cloudshell utilities and monitoring tools out of census-rm-toolbox](https://trello.com/c/XHfzQUze/1257-split-cloudshell-utilities-and-monitoring-tools-out-of-census-rm-toolbox)

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**